www/assets/js/components/FilterableFacet.tsx

```diff
@@ -80,20 +84,26 @@ function FilterableSearchFacet(props: Props) {
               placeholder={`Search ${title || ""}`}
             />
             {filterText === "" ? (
-              <i className="material-icons search-icon mt-1">search</i>
-            ) : (
               <i
-                className="material-icons clear-icon"
+                className="input-postfix-icon material-icons search-icon mt-1"
+                aria-hidden="true"
+              >
+                search
+              </i>
+            ) : (
+              <button
+                className="input-postfix-button"
+                type="button"
+                role="button"
                 onClick={() => setFilterText("")}
                 onKeyPress={e => {
                   if (e.key === "Enter") {
                     setFilterText("")
                   }
                 }}
-                tabIndex={0}
               >
-                clear
-              </i>
+                <span className="material-icons">clear</span>
+              </button>
```

Comment:
**Minor Suggestion:** `aria-hidden="true"` and a `aria-label="Clear search text"`.

The material icon names are reasonably meaningful, though.